build.gradle.kts

```diff
@@ -115,4 +122,43 @@ apiValidation {
    * Flag to programmatically disable compatibility validator
    */
   validationDisabled = false
-}
\ No newline at end of file
+}
+
+val detektAll by tasks.registering(io.gitlab.arturbosch.detekt.Detekt::class) {
```

Comment:
Should it be launched with lint on CI?

Reply:
yes, we can. but first of all we should fix all rules that I disabled.
for now we will use detekt mostly for code metrics.